packages/diracx-web/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "diracx-web",
+  "version": "0.1.0-a1",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build && rm -rf ../../out && mv out ../..",
+    "start": "next start",
+    "lint": "next lint",
+    "ts-lint": "tsc -noEmit -incremental",
+    "postinstall": "node ../../node_modules/@axa-fr/react-oidc/bin/copy-service-worker-files.mjs public"
+  },
+  "dependencies": {
+    "@axa-fr/react-oidc": "^7.22.6",
```

Comment:
The extensions code still works because the library is already a dependency of `diracx-web-components` but it's a bad practice to use transitive dependencies directly. 
I should probably add `@axa-fr/react-oidc` to the dependencies in `extensions/package.json`.